src/components/card/card.tsx

```diff
@@ -319,7 +319,7 @@ export const EuiCard: FunctionComponent<EuiCardProps> = ({
   }
 
   let optionalSelectButton;
-  if (selectable) {
+  if (selectable && description) {
     optionalSelectButton = (
       <EuiCardSelect
         aria-describedby={`${ariaId}Title ${ariaId}Description`}
```

Comment:
If earlier in the file you define a variable something like: 

```
const ariaDesc = description ? `${ariaId}Description` : '';
```

Then you can use it in place of everywhere it's used now
```suggestion
        aria-describedby={`${ariaId}Title ${ariaDesc}`}
```

Reply:
I thought of doing the same `const ariaDesc = description ? ${ariaId}Description : '';` .But I wasn't sure what value `ariaDesc`  should take i.e. if  `ariaDesc = ' ' `  is  allowed !